src/main/app/src/app/zaken/model/zaak-heropenen-gegevens.ts

```diff
@@ -0,0 +1,7 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Atos
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+export class ZaakHeropenenGegevens {
+}
```

Comment:
is deze lege class nodig, misschien even stukje commentaar dat hier later nog een reden of iets dergelijks in komt te staan